st2actions/st2actions/notifier/notifier.py

```diff
@@ -50,8 +51,8 @@
 
 LOG = logging.getLogger(__name__)
 
-ACTIONUPDATE_WORK_Q = liveaction.get_queue('st2.notifiers.work',
-                                           routing_key=publishers.UPDATE_RK)
+ACTIONUPDATE_WORK_Q = execution.get_queue('st2.notifiers.work',
```

Comment:
I'll have to test but it's probably safe to call this new something else. In a HA setup, people might do rolling restarts and this could be an issue.